modules/giapi/src/main/scala/giapi/client/gpi/GPIClient.scala

```diff
@@ -10,12 +10,13 @@ import fs2.Stream
 import giapi.client.commands.{Command, CommandResult, Configuration}
 import giapi.client.{Giapi, commands}
 import mouse.boolean._
-import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
 
 /**
   * Client for GPI
   */
-class GPIClient[F[_]](giapi: Giapi[F], ec: ExecutionContext) {
+class GPIClient[F[_]](giapi: Giapi[F]) {
+  val defaultCommandTimeout: FiniteDuration = 2000.milliseconds
```

Comment:
Can we capitalize this since it's a constant? i.e., `DefaultCommandTimeout`

I would not be entirely opposed to this being a default argument on `Command.apply`, just to remove the repetition. I don't normally like default args but it might make sense here. What do you think?

Reply:
In this case probably we don't want to have a default arg. This value is mostly the same for all GPI commands but it is not necessarily the same for other GIAPI instruments like GHOST